eps/server/common.py

```diff
@@ -41,55 +40,52 @@
 subscribed_to_headers = [False]
 are_headers_raw = [False]
 bestblockhash = [None]
-debug_fd = None
 
 #log for checking up/seeing your wallet, debug for when something has gone wrong
-def debugorlog(line, ttype):
-    timestamp = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S,%f")
-    return timestamp + " [" + ttype + "] " + line
-
-def debug(line):
-    global debug_fd
-    if debug_fd == None:
-        return
-    debug_fd.write(debugorlog(line, "DEBUG") + "\n")
-    debug_fd.flush()
-
-def log(line):
-    global debug_fd
-    line = debugorlog(line, "  LOG")
-    print(line)
-    if debug_fd == None:
-        return
-    debug_fd.write(line + "\n")
-    debug_fd.flush()
+def logger_config(logger, fmt=None, filename=None, logfilemode='w'):
+    formatter = logging.Formatter(fmt, datefmt='%Y-%m-%d %H:%M:%S')
```

Comment:
Please add ",%f" at the end of the dateformat so that the timestamps have sub-second resolution. It's useful for debugging events that happen quicker than a second.

Reply:
Will do